source/NVDAHelper.py

```diff
@@ -119,6 +120,25 @@ def nvdaControllerInternal_requestRegistration(uuidString):
 	queueHandler.queueFunction(queueHandler.eventQueue,appModuleHandler.update,pid,helperLocalBindingHandle=bindingHandle,inprocRegistrationHandle=registrationHandle)
 	return 0
 
+
+@WINFUNCTYPE(c_long, c_wchar_p, c_wchar_p)
+def nvdaControllerInternal_reportLiveRegion(text, level):
+	if not config.conf["presentation"]["reportDynamicContentChanges"]:
+		return -1
+	focus = api.getFocusObject()
+	if focus.sleepMode == focus.SLEEP_FULL:
+		return -1
+	import queueHandler
+	import speech
+	from speech.priorities import Spri
+	queueHandler.queueFunction(
+		queueHandler.eventQueue,
+		speech.speakText,

```

Comment:
How are live regions updates presented in braille? For now it looks like they aren't though it is possible I'm missing something.